sst/backward.cc

```diff
@@ -0,0 +1,42 @@
+// Pick your poison.
+//
+// On GNU/Linux, you have few choices to get the most out of your stack trace.
```

Comment:
Is there a way to automate this library installation in SST-CMake? And what happens for MacOS and non-GNU compilers?

Reply:
I will remove this file in our final integration. I needed it for debugging.

Reply:
But it is very useful for debugging atleast for SST stuff. I can try and look for automating it if you want.

Reply:
I will leave this decision up to you. Again it could be something to come back to in the future.